bdfr/__main__.py

```diff
@@ -17,6 +17,7 @@
     click.option('--authenticate', is_flag=True, default=None),
     click.option('--config', type=str, default=None),
     click.option('--disable-module', multiple=True, default=None, type=str),
+    click.option('--include-id-file', multiple=True, default=None),
```

Comment:
`--from-id-file` more suitable

Reply:
I think the current name is better, it provides an opposite to the option `--exclude-id-file`. Since the latter already exists, I think the same convention for the former would be less confusing.